contributors/design-proposals/local-storage-pv.md

```diff
@@ -0,0 +1,632 @@
+# Local Storage Persistent Volumes
+
+Authors: @msau42, @vishh
+
+This document presents a detailed design for supporting persistent local storage,
+as outlined in [Local Storage Overview](local-storage-overview.md).
+Supporting all the use cases for persistent local storage will take many releases,
+so this document will be extended for each new release as we add more features.
+
+## Goals
+
+* Allow pods to mount any local block or filesystem based volume.
+* Allow pods to mount dedicated local disks, or channeled partitions as volumes for
+IOPS isolation.
+* Allow pods do access local volumes without root privileges.
+* Allow pods to access local volumes without needing to undestand the storage
+layout on every node.
+* Persist local volumes and provide data gravity for pods.  Any pod
+using the local volume will be scheduled to the same node that the local volume
+is on.
+* Allow pods to release their local volume bindings during failure conditions, such
+as node, storage or scheduling failures.
+* Allow pods to specify local storage as part of a Deployment or StatefulSet.
+* Allow administrators to set up and configure local volumes with simple methods.
+* Do not require administrators to manage the local volumes once provisioned
+for a node.
+
+## Non-Goals
+
+* Provide data availability for a local volume beyond its local node.
+* Support the use of HostPath volumes and Local PVs on the same volume.
+
+## Background
+
+In Kubernetes, there are two main types of storage: local and remote.
```

Comment:
How about switching these around and a few other edits...
"""
In Kubernetes, there are two main types of storage: remote and local.

Remote storage is typically used with persistent volumes where the data persists beyond the lifetime of the pod.

Local storage may desirable for ephemeral use where the data persists only during the lifetime of the pod. Local storage is also desirable for distributed storage systems, such as GlusterFS and Cassandra.

The main motivations for using local storage include:

Performance:  Local SSDs can achieve higher IOPS and throughput than remote storage.

Cost: Operational costs may be reduced by leveraging existing local storage, especially in bare metal environments. Network storage can be expensive to setup and maintain, and it may not be necessary for certain applications. 

"""

Reply:
done